mfs/system.go

```diff
@@ -1,12 +1,6 @@
 // package mfs implements an in memory model of a mutable ipfs filesystem.
 //
-// It consists of four main structs:
-// 1) The Filesystem
```

Comment:
Granted, this comment wasnt entirely accurate anymore, but i would like to have something at least as informative in its place